pinot-segment-local/src/main/java/org/apache/pinot/segment/local/aggregator/DistinctCountHLLValueAggregator.java

```diff
@@ -100,6 +124,9 @@ public byte[] serializeAggregatedValue(HyperLogLog value) {
 
   @Override
   public HyperLogLog deserializeAggregatedValue(byte[] bytes) {
+    if (bytes == null || bytes.length == 0) {
+      return new HyperLogLog(_log2m);
+    }
```

Comment:
Is this needed? We don't have this special handling in other functions

Reply:
deserializeAggregatedValue is often called in getInitialAggregatedValue of some of the implementations (not just HLL or sum precision), one example is in AvgValueAggregator when the input is of type bytes, so to cover that case, I also do it here

Reply:
I don't follow. Even if it is used in `getInitialAggregatedValue()`, the input should never be null or empty. Are we trying to handle invalid input data (e.g. empty byte array)? If so, the handling should be added to `getInitialAggregatedValue()` and `applyRawValue()` instead of here